src/move/rewards/sources/emojicoin_dot_fun_rewards.move

```diff
@@ -0,0 +1,102 @@
+module rewards::emojicoin_dot_fun_rewards {
+
+    use std::signer;
+    use aptos_framework::account::{Self, SignerCapability};
+    use aptos_framework::aptos_account;
+    use aptos_framework::aptos_coin::AptosCoin;
+    use aptos_framework::coin;
+    use aptos_framework::event;
+    use aptos_framework::randomness;
+    use emojicoin_dot_fun::emojicoin_dot_fun::{Self, Swap};
+
+    /// Resource account address seed for the vault.
+    const VAULT: vector<u8> = b"Vault";
+
+    /// Flat integrator fee.
+    const INTEGRATOR_FEE_RATE_BPS: u8 = 100;
+
+    // APT amounts.
+    const REWARD_AMOUNT_IN_OCTAS: u64 = 100_000_000;
```

Comment:
This implicitly encourages people to trade as little as possible over the volume threshold (aka as many trades as possible), since it's a flat amount instead of a percentage. I'm assuming this is intended, yea?

e.g. trade ~1000 APT in volume once, 0.1 APT * 1 trade = avg 0.1 APT reward vs trade ~10 APT 100 times, aka 0.1 APT * 100 trades = avg 10 APT reward

I don't know if it really matters- just something to think about

Reply:
Correct, this is the simplest approach

If we decide we want to update later to percentage I suppose we can spin that into a new PR since the package is compatible anyways